specification/common/transactions/register/RegisterNewDelegateTransaction.yaml

```diff
@@ -0,0 +1,28 @@
+allOf:
+  - $ref: "./RegisterTransactionBase.yaml"
+  - type: object
+    required:
+      - asset
+      - recipientId
```

Comment:
Are both props `recipientId` and `senderId` required?
If yes, can you add below a description for each one?

```yaml
properties:
    # ...
    recipientId:
      type: string
      description: YOUR DESCRIPTION HERE
      example: U14236667426471084862
    senderId:
      type: string
      description: YOUR DESCRIPTION HERE
      example: U14236667426471084862
```